function/spring-aggregator-function/src/test/java/org/springframework/cloud/fn/aggregator/CustomPropsAndMongoMessageStoreAggregatorTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020-2022 the original author or authors.
+ * Copyright 2020-2024 the original author or authors.
```

Comment:
What a nice little touch [this is](https://github.com/spring-cloud/spring-functions-catalog/blob/27bffb9c4d3b6f324cc015343166835d7c98d9a5/build.gradle#L191) - thank you @artembilan